bigquery/snippets/snippet.go

```diff
@@ -166,28 +241,26 @@ func listTables(client *bigquery.Client, w io.Writer, datasetID string) error {
 
 func insertRows(client *bigquery.Client, datasetID, tableID string) error {
 	ctx := context.Background()
-	// [START bigquery_insert_stream]
+	// [START bigquery_table_insert_rows]
 	u := client.Dataset(datasetID).Table(tableID).Uploader()
 	items := []*Item{
 		// Item implements the ValueSaver interface.
-		{Name: "n1", Count: 7},
-		{Name: "n2", Count: 2},
-		{Name: "n3", Count: 1},
+		{Name: "Phred Phlyntstone", Age: 32},
+		{Name: "Wylma Phlyntstone", Age: 29},
 	}
 	if err := u.Put(ctx, items); err != nil {
 		return err
 	}
-	// [END bigquery_insert_stream]
+	// [END bigquery_table_insert_rows]
 	return nil
 }
 
 func listRows(client *bigquery.Client, datasetID, tableID string) error {
```

Comment:
Function name seems a little off. Is this sample worth keeping around at all?

Reply:
I left in the example method in and deferred the decision to cull it.  It was part of the snippets that demonstrate mapping BQ schema and results back/forth from native Go types.  I suspect the right thing to do is move it wholly to the client library docs or have a one-off example of working with Go type mappings and the ValueSaver interface.

Reply:
Okay.

We tend to be pretty liberal about deleting samples that aren't referenced from anywhere. If someone really wants it they can get it from the git history, but I'm fine leaving it in for now if you have plans to migrate it.